test/integration/.eslintrc.yml

```diff
@@ -0,0 +1,7 @@
+extends: '../.eslintrc.yml'
+env:
+  jquery: true
+globals:
+  browser: writable
+rules:
+  strict: ['error', 'global']
```

Comment:
allows the use of `use strict` at the top of the file, instead of requiring it to be in function scope